virt/getting_started/virt-web-console-overview.adoc

```diff
@@ -510,7 +510,7 @@ You configure a virtual machine on the *VirtualMachine details* page.
 [id="virtualmachine-details-overview_virt-web-console-overview"]
 ==== Overview tab
 
-The *Overview* tab displays resource usage, alerts, and configuration information.
+The *Overview* tab displays resource usage, alerts, configuration information, and live migration progress.
```

Comment:
The live migration progress only shows in Metrics tab, not Overview tab.

Reply:
Thanks! I've made the correction.